spec/serializer_spec.rb

```diff
@@ -364,6 +364,13 @@ def serialize_primary(object, options = {})
       error = JSONAPI::Serializer::AmbiguousCollectionError
       expect { JSONAPI::Serializer.serialize(posts) }.to raise_error(error)
     end
+
+    it "raises error if include is not named correctly" do
+      post = create(:post)
+      error = JSONAPI::Serializer::InvalidIncludeError
+      expect { JSONAPI::Serializer.serialize(post, :include => ['long_comments']) }.to raise_error(error)
```

Comment:
For consistency with the rest of this codebase, please use `include: ...` style syntax instead of the hashrocket style.